kirkstone-next.xml

```diff
@@ -26,11 +26,13 @@
   <project remote="python2" upstream="kirkstone"   name="meta-python2"            path="sources/meta-python2"/>
 
   <project remote="hmssh" revision="kirkstone" name="meta-hostmobility-bsp" path="sources/meta-hostmobility-bsp" />
-  <project remote="hmssh" revision="master" name="meta-mobility-poky-distro" path="sources/meta-mobility-poky-distro"/>
+  <project remote="hmssh" revision="Browser-update" name="meta-mobility-poky-distro" path="sources/meta-mobility-poky-distro"/>
```

Comment:
You'd want to change this before merge, right?

Reply:
yes